bootstrap.js

```diff
@@ -0,0 +1,142 @@
+import { readFile } from './fs/promises.js'
+import { createWriteStream } from './fs/index.js'
+import { PassThrough } from './stream.js'
+import { createDigest } from './crypto.js'
+import { EventEmitter } from './events.js'
+
+async function* streamAsyncIterable(stream) {
+  const reader = stream.getReader();
```

Comment:
can you run standard linter on this file? some of it is standard-ish, some of it not.